src/test/java/com/emc/object/s3/AbstractS3ClientTest.java

```diff
@@ -102,6 +102,7 @@ protected static S3Config s3ConfigFromProperties() throws Exception {
         String secretKey = TestConfig.getPropertyNotEmpty(props, TestProperties.S3_SECRET_KEY);
         URI endpoint = new URI(TestConfig.getPropertyNotEmpty(props, TestProperties.S3_ENDPOINT));
         boolean enableVhost = Boolean.parseBoolean(props.getProperty(TestProperties.ENABLE_VHOST));
+        boolean isSmartClient = Boolean.parseBoolean(props.getProperty(TestProperties.IS_SMART_CLIENT));
```

Comment:
Should be ENABLE_SMART_CLIENT?

Reply:
Have changed the property name to ENABLE_SMART_CLIENT